zict/tests/test_file.py

```diff
@@ -43,11 +43,12 @@ def test_memmap_implementation(fn):
     z = File(fn, memmap=True)
     assert not z
 
-    z["x"] = b"123"
+    mv = memoryview(b"123")
```

Comment:
Previously we were testing that we could add `bytes` to the mapping and get out a `memoryview`. Now we're doing `memoryview` in and `memoryview` out. Is the `bytes` case intended to be supported?

Reply:
We can try both if you like. Either should work as both `writelines` & `write` can handle any [Python Buffer Protocol object]( https://docs.python.org/3/c-api/buffer.html )

That particular code path is currently unaffected by the `memmap` flag